### ISSUE TITLE
Cannot run locally.

### ISSUE THREAD

**Issue by L16H7** [suddi/suddi.github.io#14]
First, uglifyJs is not working correctly.  Got this error.

> ERROR in bundle.js from UglifyJs
> Unexpected token: name (urlParts) [(webpack)-dev-server/client?http:/localhost:8080:24,0][bundle.js:32725,4]
> 
So, I commented plugins in webpack.config.js. It compiled successfully. I got this error at localhost:8080

> Uncaught TypeError: Cannot read property 'location' of undefined 
Thank you.

**Comment by suddi** [comment]
Thank you @01001100MK for letting me know on this issue.

The problem was caused by the following problem: https://github.com/webpack/webpack-dev-server/issues/1101

I have corrected it here: https://github.com/suddi/suddi.github.io/commit/54703c4150753bce05abb81f45819ee2ab82ad3b using the workaround solution by downgrading the version of `webpack-dev-server`.

Let me know if you run into any further issues.
